src/include/Token.h

```diff
@@ -99,4 +99,14 @@ struct Token {
             // TODO: other cases
         }
     }
+    unsigned char getUnOpcode() {
+        switch (type) {
+            case TokenType::Plus:
+                return Opcode::POS;
```

Comment:
Do we want to have line breaks after one line cases? It looks like we don't above. I don't mind either way, however, I think we should be consistent

Reply:
Good question. I was avoiding it in `tokTypeToString()` in particular because it's a super-long function and I don't want it to be twice that length 😛 I also think it's a *bit* more readable since you can just scan directly to the case you want (you don't have to "skip" over the `return` statements). But that being said I don't think @CoffeeTableEspresso's way looks bad either.